CHANGES.md

```diff
@@ -1,3 +1,80 @@
+### NUnit 3.10 - March 12, 2018
+
+This release adds a .NET Standard 2.0 version of the framework which re-enables
+most of the features that have been missing in our earlier .NET Standard builds
+like parallelism, timeouts, directory and path based asserts, etc. It also contains
+numerous bug fixes and smaller enhancements. We've improved our XML docs,
+fixed performance issues and added more detail to Multiple Asserts.
+
+This release also contains source linked PDB files allowing developers to debug
```

Comment:
Technically GitLink is not the same mechanism as Source Link (sourcelink.json in the PDB) so that might cause people to think the wrong thing.

Reply:
I am okay to update if the user behavior is likely to be different or confusing. What text would you suggest? Source linked with GitLink?

Reply:
The terminology I've seen is source indexing and source-indexed PDBs. That's the name of the TFS build task and I see that at https://github.com/GitTools/GitLink.